h2o-core/src/main/java/water/Paxos.java

```diff
@@ -71,23 +71,7 @@ static synchronized int doHeartbeat( H2ONode h2o ) {
       ListenerService.getInstance().report("different_cloud", h2o);
       return 0;
     }
-
-
-    // Update manual flatfile in case of flatfile is enabled
-    if (H2O.isFlatfileEnabled()) {
-      if (!H2O.ARGS.client && h2o._heartbeat._client && !H2O.isNodeInFlatfile(h2o)) {
-        // A new client was reported to this node so we propagate this information to all nodes in the cluster, to this
```

Comment:
Since Client's flatfile contains all the worker nodes we can be sure client will start pinging all of the worker nodes